components/org.wso2.carbon.identity.api.server.certificate.validation.management/org.wso2.carbon.identity.api.server.certificate.validation.management.v1/src/main/java/org/wso2/carbon/identity/api/server/certificate/validation/management/v1/util/ModelMapperUtil.java

```diff
@@ -0,0 +1,123 @@
+import org.wso2.carbon.identity.api.server.certificate.validation.management.v1.model.Validator;
+import org.wso2.carbon.identity.api.server.certificate.validation.management.v1.model.Validators;
+import org.wso2.carbon.identity.x509Certificate.validation.model.CACertificateInfo;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Utility class for Certificate Validation Management endpoint.
+ */
+public class ModelMapperUtil {
+
+    private ModelMapperUtil() {
+
+    }
+
+    /**
+     * Map Validator model to API model.
+     *
+     * @param validators List of validators.
+     * @return List of API validators.
+     */
+    public static Validators mapValidatorsToApiModel
+    (List<org.wso2.carbon.identity.x509Certificate.validation.model.Validator> validators) {
+
+        List<String> validatorList = new ArrayList<>();
+        for (org.wso2.carbon.identity.x509Certificate.validation.model.Validator validator : validators) {
+            validatorList.add(validator.getDisplayName());
+        }
+        return new Validators().validators(validatorList);
+    }
+
+    /**
+     * Map Validator model to API model.
+     *
+     * @param validator Validator.
+     * @return API validator.
+     */
+    public static Validator mapValidatorToApiModel
+    (org.wso2.carbon.identity.x509Certificate.validation.model.Validator validator) {
+
+        Validator apiValidator = new Validator();
+        apiValidator.setEnable(validator.isEnabled());
+        apiValidator.setFullChainValidation(validator.isFullChainValidationEnabled());
+        apiValidator.setPriority(validator.getPriority());
+        apiValidator.setRetryCount(validator.getRetryCount());
+        return apiValidator;
+    }
+
+    /**
+     * Map API model to Certificate Validator object.
+     *
+     * @param validator API Validator.
+     * @return Certificate Validator object.
+     */
+    public static org.wso2.carbon.identity.x509Certificate.validation.model.Validator
+    mapApiModelToCertificateValidatorObject(Validator validator, String validatorName) {
+
+        org.wso2.carbon.identity.x509Certificate.validation.model.Validator certificateValidator =
+                new org.wso2.carbon.identity.x509Certificate.validation.model.Validator();
+        certificateValidator.setDisplayName(validatorName);
+        certificateValidator.setEnabled(validator.getEnable());
+        certificateValidator.setFullChainValidationEnabled(validator.getFullChainValidation());
+        certificateValidator.setPriority(validator.getPriority());
+        certificateValidator.setRetryCount(validator.getRetryCount());
+        return certificateValidator;
+    }
+
+    /**
+     * Map CA Certificates model to API model.
+     *
+     * @param caCertificateInfoList List of CA Certificates.
+     * @return API CA Certificates.
+     */
+    public static CACertificates mapCACertificatesToApiModel(List<CACertificateInfo> caCertificateInfoList) {
+
+        List<CACertificate>
```

Comment:
Use Java Streams for readibility

Reply:
Fixed with https://github.com/wso2/identity-api-server/pull/834/commits/a768a6bfccc225291c096836167adc0f03fda24c